tests/test_fluxFromABMag.py

```diff
@@ -56,6 +58,20 @@ def testBasics(self):
                     abMagErr, abMag)
                 self.assertAlmostEqual(fluxErr, fluxErrRoundTrip)
 
+    def testVector(self):
+        flux = np.array([1.0, 210.0, 3210.0, 43210.0, 543210.0])
+        abMag = afwImage.abMagFromFlux(flux)
+        np.testing.assert_array_almost_equal(abMag, refABMagFromFlux(flux))
```

Comment:
I would suggest using `self.assertFloatsAlmostEqual` for your comparisons here: it's more strict that `np.testing`.

https://developer.lsst.io/coding/python_testing.html#special-asserts